src/main/resources/application.yml

```diff
@@ -44,6 +44,7 @@ spring:
 image.recognition:
   region: eu-west-2
   aws.collection.id: dps-offender-images
+  face.similarity.threshold: 96.0
```

Comment:
Analysis of this running in preprod has shown 95% was too low - now running in preprod every day so will continue to refine this threshold when we get more data.